features/mouse-events.yml.dist

```diff
@@ -0,0 +1,191 @@
+# Generated from: mouse-events.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2015-07-29
+  baseline_high_date: 2018-01-29
+  support:
+    chrome: "2"
```

Comment:
Without doing any research I'm confident this should be the earliest release of all browsers. Mouse events are ancient, and there's no chance that they were so broken in any browser released since 2003 (Safari 1) that we should consider them not supported.

At the same time I don't want to spend any time researching the small differences here. How about just `compute_from: api.MouseEvent` and a comment/TODO?